### DISCUSSION TITLE
Components/libraries/functions suggestions

### DISCUSSION THREAD

**Discussion by steven-tey** [steven-tey/precedent#15] [Ideas]
What other libraries/functions/components that you'd like to see added to this template?

E.g. Radix's [navigation menu](https://www.radix-ui.com/docs/primitives/components/navigation-menu) could be a great addition – but we'll need to think about how to make it mobile-compatible :)

**Comment by schonert** [comment]
Great stack!

Been an eager follower Radix and Stitches from the early days - and I feel like the team has lost quite some momentum after they were purchased by WorkOS. Not pointing or blaming - just observing. 

I could be hesitant relying too much on Radix since the progress is done by the team and not so much by the community. If their time gets allocated internally the project might stall. They are incredibly talented and creating components with the vision and quality they present raises the bar for contributors quite a lot. 

One could hope that this project could push more projects with Radix in production and engage outside commitment. 

Just my two cents.

**Reply by steven-tey** [reply]
Oooh, very interesting observation here! Do you know any other libs like Radix that has the same level of a11y support that you would replace it with?

**Reply by schonert** [reply]
Nope - not stumbled upon anything in that league. There are a few approaches with hooks, but the primitive approach seems more intuitive. 

Reading your comment made me wonder if I might have dramatized the impact of the project stalling. If the basics are nailed and your setup worked from the point it stalled - I guess everything is still good. Potential security risks would be minimal I guess.

**Comment by alex289** [comment]
How about adding [nprogress](https://ricostacruz.com/nprogress/). I really like this loading indicator and think this would suit well in this project.

[How to use nprogress in Nextjs](https://learnjsx.com/category/4/posts/nextjs-nprogress)
